helpers.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash -e
+# Helpers for functionality that can be shared within release-tools
+
+function retry() {
```

Comment:
I'm on the fence here.

When we start doing helpers and functions, we should probably move out of bash and into Python.  I realize there is existing code in here to provide small helpers here and there, but it is a slippery slope.  This seems pretty small and simple, but I worry about how it might be extended (or broken, without test coverage).

Also:  is it really necessary?

Reply:
In essence, it's not necessary to make it into a shared helper. My only concern with moving this code into Python is that it'd basically end up being a python retry function around a `subprocess.check_call` which I don't really think buys us much.

Reply:
On the other hand, we could import things like charm-build directly and use that via Python. I'm not sure that the benefit is enough to warrant the time to do that though